### DISCUSSION TITLE
Background image cannot be displayed

### DISCUSSION THREAD

**Discussion by SardorKamoliddinov** [tailwindlabs/tailwindcss#13799] [Help]
I can not displayed background image with tailwind css

**Comment by patrick521f** [comment]
Hello, @SardorKamoliddinov 

Here’s a step-by-step guide to help you set up a background image:

Add the Background Image Class: Use the bg-[url('path/to/image')] syntax to add your background image.
Ensure the Image Covers the Element: Apply the bg-cover class to make sure the image covers the entire element.
Center the Image: Use the bg-center class to center your background image within the element.
No Repeat: To prevent the image from repeating, you can use the bg-no-repeat class.
Set Custom Sizes in Config: If you need specific sizes, you can customize your tailwind.config.js file to include custom background sizes.
Here’s an example of how your HTML element with Tailwind classes might look:

<div class="bg-[url('/path/to/your/image.jpg')] bg-cover bg-center bg-no-repeat h-full w-full">
  
</div>

Make sure the path to your image is correct and that your Tailwind configuration allows for custom values if you’re using them. If you continue to face issues, it could be related to other CSS rules that might be overriding your Tailwind classes, or there might be a configuration issue in your tailwind.config.js file.


I hope this info is helpful to you.
[aarp-mahjongg.com](https://www.aarp-mahjongg.com)
Best Regard,
patrick521f

**Comment by SardorKamoliddinov** [comment]
Hi @patrick521f 

Thank you for your answer

In the past I wrote this syntax but image sometimes get out sometimes do not get out

I will try again like similar with your advice 

Best Regard, 
SardorbekKamoliddinov
